Packs/ApiModules/Scripts/IAMApiModule/IAMApiModule.py

```diff
@@ -330,7 +330,7 @@ def get_user(self, client, args):
                     action=IAMActions.GET_USER,
                     active=user_app_data.is_active,
                     iden=user_app_data.id,
-                    email=email,
+                    email=user_profile.get_attribute('email'),
```

Comment:
```suggestion
                    email=user_profile.get_attribute(self.attr),
```

Reply:
Here its ok, we want to return the email anyways to the Human readable entry